api/app/src/main/kotlin/packit/AppConfig.kt

```diff
@@ -16,9 +16,8 @@ class AppConfig(private val props: PackitProperties = properties)
     val dbUrl: String = propString("db.url")
     val dbUser: String = propString("db.user")
     val dbPassword: String = propString("db.password")
-    val authBasicSecret: String = propString("auth.basic.secret")
+    val authJWTSecret: String = propString("auth.jwt.secret")
```

Comment:
This relates to generating the JWT token so this seemed like a less confusing name...